test/generate/test_generate.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import time
+from typing import Optional
+
+import torch
+import torch.distributed.checkpoint as dcp
+
+from generation import generate
```

Comment:
I'm not sure: would it be better to do this or `from test.generation import generate`?